web/src/App.jsx

```
@@ -843,17 +843,22 @@ const MessageComposer = memo(({ user, db, selectedThread, lineInboxMode, activeL
           {body.length > 0 && (
             <div
               id="message-char-count"
+              role="status"
+              aria-live="polite"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)